packages/analytics/metric-cards/src/components/MetricCardContainer.vue

```diff
@@ -33,10 +33,13 @@
         :key="index"
         v-bind="formatCardValues(card)"
         :card-size="cardSize"
+        :card-type="(card.cardType as MetricCardType)"
```

Comment:
Out of curiosity, is this cast still needed?

Reply:
Yup 😬 

<img width="974" alt="Screenshot 2023-10-27 at 8 57 35 AM" src="https://github.com/Kong/public-ui-components/assets/103061463/18472fcd-6c95-4fa2-9fb7-3aadc8d69665">

Reply:
This means you're coercing a possibly undefined value into a prop that expects it to be defined.  Somewhere, you need to have a default value.

Reply:
I'll default it to the Generic Count card type then